src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs').promises;
```

Comment:
`.promises` is the old way of importing fs promises api. [Current docs](https://nodejs.org/docs/latest-v18.x/api/fs.html#promises-api) suggest doing it like this:
```suggestion
const fs = require('node:fs/promises');
```

Not an issue, but this might not work in future node.js versions